### PR TITLE
feat: accordedRight to be an array in createEntity and createEntityAs…

### PR DESCRIPTION
**Pull request by yousefhany77**
…Action

## Screenshots
Screenshot 2024-09-16 at 10 36 22 PM

Screenshot 2024-09-16 at 10 36 33 PM

### CODE REVIEW

File: apps/web/src/components/Entities/CreateEntityFlow/Review/Review.tsx [hunk 1]
```diff
@@ -145,6 +148,7 @@ const Review = ({ showNavigation = true }: { showNavigation?: boolean }): JSX.El
 
   return (
     <FlexBox width={`${deviceWidth.tablet}px`} $gap={10} $alignItems='stretch'>
+      here
```
Comment (Zach-ixo) [inline apps/web/src/components/Entities/CreateEntityFlow/Review/Review.tsx]: This random here? is it intended @yousefhany77

File: apps/web/src/hooks/createEntity.ts [hunk 2]
```diff
@@ -544,6 +544,7 @@ export function useCreateEntity(): TCreateEntityHookRes {
         relayerNode,
         controller = [],
       } = payload
+      console.log("🚀 ~ useCreateEntity ~ payload:", payload)
```
Comment (Zach-ixo) [inline apps/web/src/hooks/createEntity.ts]: Log can go

File: apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx [hunk 3]
```diff
@@ -55,6 +55,17 @@ const EditProperty: React.FC = (): JSX.Element => {
   }, [])
 
   const PropertiesFormProps = useMemo(() => {
+    console.log('🚀 ~ editEntity:', editEntity?.accordedRight)
```
Comment (Zach-ixo) [inline apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx]: Log can go

File: apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx [hunk 4]
```diff
@@ -55,6 +55,17 @@ const EditProperty: React.FC = (): JSX.Element => {
...
+    const obj = Array.isArray(editEntity?.accordedRight)
```
Comment (Zach-ixo) [inline apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx]: And maybe rename obj to how accordedRights are formed here for better cognitive complexity

File: apps/web/src/components/Entities/CreateEntityFlow/Review/Review.tsx (hunk 1 above)
Comment (yousefhany77) [reply apps/web/src/components/Entities/CreateEntityFlow/Review/Review.tsx]: will remove it. forgot to remove it

File: apps/web/src/hooks/createEntity.ts (hunk 2 above)
Comment (yousefhany77) [reply apps/web/src/hooks/createEntity.ts]: will remove it. forgot to remove it

File: apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx (hunk 4 above)
Comment (yousefhany77) [reply apps/web/src/screens/CurrentEntity/Dashboard/components/EditProperty.tsx]: ok

**Review by Zach-ixo — APPROVED** [review]

**Pull request merged**